### DISCUSSION TITLE
Question about the current version (v0.10.3)

### DISCUSSION THREAD

**Discussion by turbosonics** [MDIL-SNU/SevenNet#148] [Q&A] [answered]
I remember, for 2024 Summer, multi-GPU SevenNet LAMMPS MD simulation was unable to correctly calculate pressure and stress from NPT ensemble and D3 dispersion. So, it was recommended to use single GPU calculation or ignore pressure and stress from multi-GPU simulations. 

But now I don't see those limitations in the official documentations. Are these limitations lifted? Can I test multi-GPU SevenNet LAMMPS MD simulations with NPT with solid-liquid interaction with new version's pre-trained model?

**Comment by YutackPark** [comment, marked as answer]
Yes. e3gnn/parallel now computes pressure correctly.

However, e3gnn/parallel + D3 is not possible as D3 does not support
parallel. Long cutoff of D3 makes the problem complicate.

If you're willing to use e3gnn/parallel to your system, I recommend you to
firstly check energy, force, stress of e3gnn/parallel and e3gnn before move
on. Although we test it for 1, 2, and 4 processors with PBC on and off (+
upto 32 GPUs in the papar), MD has too many possible edge cases.

2024년 12월 17일 (화) 오전 4:18, turbosonics ***@***.***>님이 작성:
